eve/workers/end2end/kind-config.yaml

```diff
@@ -3,6 +3,22 @@ kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
```

Comment:
This file is kind of the same used in `bootstrap-kind.sh` correct?

If so, shouldn't we remove it? feels like we're duplicating code but I may be wrong.